src/shared/components/selected-species/SpeciesLozenge.tsx

```diff
@@ -56,11 +65,97 @@ const SpeciesLozenge = (props: Props) => {
   return (
     <button className={componentClasses} {...otherProps}>
       <div className={styles.inner}>
-        <span className={styles.name}>{displayName}</span>
-        <span className={styles.assembly}>{props.species.assembly.name}</span>
+        <LozengeContent
+          species={species}
+          displayOption={speciesNameDisplayOption}
+        />
       </div>
     </button>
   );
 };
 
+type DisplayStringProps = {
```

Comment:
Naming... A component is not a string.

It's probably the easiest just to declare the types inline, without naming the type:

```
const LozengeContent = (props: {
  species: CommittedItem;
  displayOption: SpeciesNameDisplayOption;
}) => {
  /* component body */
}
```